cmd/bbolt/main.go

```diff
@@ -58,6 +58,9 @@ var (
 
 	// ErrNotEnoughArgs is returned with a cmd is being executed with fewer arguments.
 	ErrNotEnoughArgs = errors.New("not enough arguments")
+
+	// ErrTooManyArgs is returned with a cmd is being executed with more arguments than required.
+	ErrTooManyArgs = errors.New("too many arguments")
```

Comment:
With the Cobra migration, I think this file will eventually disappear. Would moving it to `command_check.go` or `command_root.go` make sense? Maybe we should even consider creating an `error.go` file?

Reply:
done 👍🏽